googleapis/google-cloud-node
# migrate code from googleapis/nodejs-contact-center-insights

- feat: initial stub of library
- feat: initial generation of templated files
- feat: add initial samples (#2)
- fix(deps): require google-gax v2.17.0 (#6)
- chore(deps): update dependency sinon to v11 (#4)
- chore(deps): update dependency mocha to v9 (#3)
- build: auto-approve renovate-bot PRs for minor updates (#1145) (#8)
- chore: release 1.0.0 (#1)
- chore: release 1.0.0 (#9)
- fix(deps): google-gax v2.17.1 (#11)
- chore: release 1.0.1 (#12)
- build(nodejs): correct npm artifact name (#15)
- chore: release 1.0.2 (#18)
- chore(deps): update dependency @types/mocha to v9 (#17)
- feat: update CCAI Insights protos. change nesting of Conversation.Transcript.Participant to ConversationParticipant and remove AnnotationBoundary.time_offset (#19)
- chore: release 1.1.0 (#21)
- chore(nodejs): update client ref docs link in metadata (#23)
- feat: add new issue model API methods
- chore: release 1.2.0 (#26)
- fix(build): migrate to using main branch (#28)
- fix(deps): google-gax v2.24.1 (#30)
- chore: release 1.2.1 (#29)
- feat: turns on self-signed JWT feature flag (#35)
- chore: release 1.3.0 (#36)
- feat: display_name is the display name for the assigned issue (#39)
- chore: release 1.4.0 (#40)
- chore: add CCAI team to samples CODEOWNERS (#42)
- samples: get operation (#52)
- samples: create conversation, create analysis, export data to BigQuery (#34)
- samples: enable pubsub notifications (#48)
- feat: filter is used to filter conversations used for issue model training feat: update_time is used to indicate when the phrase matcher was updated (#55)
- samples: create topic model (#49)
- chore: release 1.5.0 (#58)
- samples: set project-level TTL (#51)
- samples: create custom highlights (#53)
- samples: remove sample links from linkinator config (#60)
- feat: add metadata from dialogflow related to transcript segment, sentiment data for transcript segment (#64)
- chore: release 1.6.0 (#65)
- feat: deprecate issue_matches docs: if conversation medium is unspecified, it will default to PHONE_CALL (#68)
- chore: release 1.7.0 (#69)
- build(node): update deps used during postprocessing (#1243) (#70)
- chore: fix the wrong post processor image (#72)
- chore(deps): update dependency @types/node to v16 (#71)
- chore(deps): update dependency sinon to v12 (#73)
- feat: Add ability to update phrase matchers feat: Add issue model stats to time series feat: Add display name to issue model stats (#74)
- chore: release 1.8.0 (#75)
- docs(samples): add example tags to generated samples (#76)
- feat: remove feature flag disable_issue_modeling (#78)
- feat: Add WriteDisposition to BigQuery Export API (#80)
- chore: release 1.9.0 (#81)
- build: add generated samples to .eslintignore (#82)
- docs(node): support "stable"/"preview" release level (#1312) (#85)
- chore: add api_shortname and library_type to repo metadata (#84)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#87)
- test(nodejs): remove 15 add 16 (#1322) (#90)
- feat: new API for the View resource (#89)
- docs: Clarify comments of ConversationView enum (#93)
- chore(main): release 1.10.0 (#94)
- chore: update v2.12.0 gapic-generator-typescript (#95)
- chore(deps): update dependency sinon to v13 (#97)
- docs(nodejs): version support policy edits (#1346) (#99)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#100)
- chore: update copyright to 2022 (#101)
- chore: update v2.14.2 gapic-generator-typescript (#105)
- feat: promote to stable (#106)
- chore(main): release 1.11.0 (#107)
- build(node): update client library version in samples metadata (#1356) (#116)
- chore(deps): update dependency sinon to v14 (#121)
- build!: update library to use Node 12 (#122)
- chore(main): release 2.0.0 (#124)
- docs: Updating comments (#128)
- fix(deps): update dependency @google-cloud/bigquery to v6 (#127)
- fix(deps): update dependency @google-cloud/pubsub to v3 (#126)
- chore(main): release 2.0.1 (#129)
- chore(deps): update dependency jsdoc-region-tag to v2 (#132)
- chore(deps): update dependency jsdoc-fresh to v2 (#131)
- feat: add LRO to API list
- chore(main): release 2.1.0 (#134)
- chore(deps): update dependency linkinator to v4 (#135)
- docs: Updating comments (#136)
- fix: change import long to require (#138)
- fix: do not import the whole google-gax from proto JS (#1553) (#141)
- fix: allow passing gax instance to client constructor (#137)
- chore(main): release 2.1.1 (#139)
- chore(deps): update dependency uuid to v9 (#142)
- fix: preserve default values in x-goog-request-params header (#143)
- test: use fully qualified request type name in tests (#158)
- fix(deps): use google-gax v3.5.2 (#162)
- chore(deps): update dependency @types/node to v18 (#160)
- chore(deps): update dependency jsdoc to v4 (#164)
- fix: regenerated protos JS and TS definitions (#165)
- chore(main): release 2.1.2 (#144)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
